includes/utils.inc

```diff
@@ -287,10 +287,10 @@ function wf_crm_get_matching_rules($contact_type) {
   static $rules;
   $contact_type = ucfirst($contact_type);
   if (!$rules) {
-    $rules = array_fill_keys(array('Individual', 'Organization', 'Household'), array());
-    $dao = CRM_Core_DAO::executeQuery('SELECT * FROM civicrm_dedupe_rule_group');
-    while ($dao->fetch()) {
-      $rules[$dao->contact_type][$dao->id] = $dao->title;
+    $rules = array_fill_keys(['Individual', 'Organization', 'Household'], []);
+    $values = wf_crm_apivalues('RuleGroup', 'Get', []);
```

Comment:
Typically the api action is all lowercase, and this function's 3rd param already defaults to `[]` so I'd write this line as

    $values = wf_crm_apivalues('RuleGroup', 'get');

Reply:
Hoi Klaas: I tried to commit Coleman's suggestion (a new button popped up in GitHub) - but I don't have permissions - which is odd b/c I know I can do that from the command line.

Reply:
Hi @KarinG, I see you already solved it (I was on holiday in Italy without laptop)